{
  "info": {
    "author": "Forgeflow, Odoo Community Association (OCA)",
    "author_email": "support@odoo-community.org",
    "bugtrack_url": null,
    "classifiers": [
      "Framework :: Odoo",
      "Framework :: Odoo :: 13.0",
      "License :: OSI Approved :: GNU Affero General Public License v3",
      "Programming Language :: Python"
    ],
    "description": "",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "https://github.com/OCA/partner-contact",
    "keywords": "",
    "license": "AGPL-3",
    "maintainer": "",
    "maintainer_email": "",
    "name": "odoo13-addon-partner-exception",
    "package_url": "https://pypi.org/project/odoo13-addon-partner-exception/",
    "platform": null,
    "project_url": "https://pypi.org/project/odoo13-addon-partner-exception/",
    "project_urls": {
      "Homepage": "https://github.com/OCA/partner-contact"
    },
    "release_url": "https://pypi.org/project/odoo13-addon-partner-exception/13.0.1.0.1.dev2/",
    "requires_dist": [
      "odoo13-addon-base-exception",
      "odoo (<13.1dev,>=13.0a)"
    ],
    "requires_python": ">=3.5",
    "summary": "Custom exceptions on partner contact",
    "version": "13.0.1.0.1.dev2",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 17132563,
  "releases": {
    "13.0.1.0.1.dev2": [
      {
        "comment_text": "",
        "digests": {
          "blake2b_256": "5b434277880712831fb12b9ea4c7fb929dbb079e92c4a0a3b02569fefbae21a8",
          "md5": "9b3d29b2d03b89f68659dd462111f83f",
          "sha256": "6cd50c07e62a20fccf2efe23e4cd3a12f9a09182fea5be4bd8368ad38f21a68c"
        },
        "downloads": -1,
        "filename": "odoo13_addon_partner_exception-13.0.1.0.1.dev2-py3-none-any.whl",
        "has_sig": false,
        "md5_digest": "9b3d29b2d03b89f68659dd462111f83f",
        "packagetype": "bdist_wheel",
        "python_version": "py3",
        "requires_python": ">=3.5",
        "size": 14108,
        "upload_time": "2023-03-02T18:28:39",
        "upload_time_iso_8601": "2023-03-02T18:28:39.052232Z",
        "url": "https://files.pythonhosted.org/packages/5b/43/4277880712831fb12b9ea4c7fb929dbb079e92c4a0a3b02569fefbae21a8/odoo13_addon_partner_exception-13.0.1.0.1.dev2-py3-none-any.whl",
        "yanked": false,
        "yanked_reason": null
      }
    ]
  },
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "5b434277880712831fb12b9ea4c7fb929dbb079e92c4a0a3b02569fefbae21a8",
        "md5": "9b3d29b2d03b89f68659dd462111f83f",
        "sha256": "6cd50c07e62a20fccf2efe23e4cd3a12f9a09182fea5be4bd8368ad38f21a68c"
      },
      "downloads": -1,
      "filename": "odoo13_addon_partner_exception-13.0.1.0.1.dev2-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "9b3d29b2d03b89f68659dd462111f83f",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": ">=3.5",
      "size": 14108,
      "upload_time": "2023-03-02T18:28:39",
      "upload_time_iso_8601": "2023-03-02T18:28:39.052232Z",
      "url": "https://files.pythonhosted.org/packages/5b/43/4277880712831fb12b9ea4c7fb929dbb079e92c4a0a3b02569fefbae21a8/odoo13_addon_partner_exception-13.0.1.0.1.dev2-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}